### ISSUE TITLE
insmod: ERROR: could not insert module mt7610u.ko: Unknown symbol in module

### ISSUE THREAD

**Issue by mathew832** [ulli-kroll/mt7610u#70]
It seems to compile correctly on 4.15.0-20-generic but when I insmod I get this error. 

Dmesg output:
[ 1141.664480] mt7610u: Unknown symbol cfg80211_inform_bss_frame_data (err 0)
[ 1141.664521] mt7610u: Unknown symbol cfg80211_scan_done (err 0)
[ 1141.664554] mt7610u: Unknown symbol regulatory_hint (err 0)
[ 1141.664589] mt7610u: Unknown symbol wiphy_new_nm (err 0)
[ 1141.664611] mt7610u: Unknown symbol wiphy_register (err 0)
[ 1141.664649] mt7610u: Unknown symbol wiphy_unregister (err 0)
[ 1141.664694] mt7610u: Unknown symbol ieee80211_channel_to_frequency (err 0)
[ 1141.664720] mt7610u: Unknown symbol ieee80211_frequency_to_channel (err 0)
[ 1141.664741] mt7610u: Unknown symbol cfg80211_connect_done (err 0)
[ 1141.664755] mt7610u: Unknown symbol wiphy_free (err 0)

**Comment by Patryk27** [comment]
For future reference: try to run `sudo modprobe cfg80211` before executing `insmod`.

**Comment by M1NDWARE** [comment]
@Patryk27 did like you said 

no crash but adapter is in unmanged mode 
dmesg:

[  458.130123] cfg80211: Loading compiled-in X.509 certificates for regulatory database
[  458.130771] cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
[  503.932640] 
               
               === pAd = 000000001790067c, size = 888216 ===

[  503.932757] <-- RTMPAllocTxRxRingMemory, Status=0
[  503.932812] <-- RTMPAllocAdapterBlock, Status=0
[  503.932933] ==>mt7610u_chip_onoff(): OnOff:1, Reset= 0, pAd->WlanFunCtrl:0x0, Reg-WlanFunCtrl=0xff000002
[  503.933200] mac_rev = 0x76502000
[  503.933550] mac_rev = 0x76502000
[  503.933729] 80211> RFICType = 3
[  503.933732] 80211> Number of channel = 44
[  503.933734] 80211> Number of rate = 12
[  503.933736] 80211> CurTxPower = 0 dBm
[  503.933738] 80211> TxStream = 0
[  503.933851] crda> requlation requestion by core: 
[  503.933853] 00
[  503.934046] 80211> CFG80211_Register
[  503.934642] usbcore: registered new interface driver mt7610u
[  504.006703] show_signal_msg: 11 callbacks suppressed
[  504.006707] ModemManager[814]: segfault at 0 ip 0000559d92372659 sp 00007ffede7311e0 error 4 in ModemManager[559d92337000+123000]

**Comment by Patryk27** [comment]
Yeah, I've personally failed to use this driver too, unfortunately - it connected to the network, but no further packets were sent (on the newest Ubuntu Server 18.04, IIRC). I believe `network-manager` also liked to suddenly segfault.

**Comment by danyalejandro** [comment]
Can confirm this, been using this driver without major problems until the latest kernel, Now I'm unable to insmod and thus have no internet anymore.

Any hacks to make this work again?
